Sources/SourceControl/GitRepository.swift

```diff
@@ -922,3 +963,153 @@ public struct GitCloneError: Error, CustomStringConvertible, DiagnosticLocationP
         return "\(self.message):\n\(output)"
     }
 }
+
+public enum GitFetchProgress: FetchProgress {
```

Comment:
does this and all its method need to be public?

Reply:
GitFetchProgress -> GitProgressParser

Reply:
done 5a5ebc6